src/index.js

```diff
@@ -218,6 +218,15 @@ export class BrowserHistory extends History {
     return historyIndex;
   }
 
+  /**
+   * Move to a specific position in the navigation history.
+   * @param movement The amount of steps, positive or negativ, to move.
+   * @returns The current index.
+   */
+  go(movement: number): void {
```

Comment:
We would need an abstract method for the base class `History` in `aurelia-history`. It would be an error TS folks without it. This is where the implementation is only.

Reply:
Am on it.

Reply:
Already :heart: your PRs 👍